Feature.02/update

Link-Spirit opened the pull request:
Update Selection created ; Allowing Task to be updated. I need to implement ID checker to ensure if valid or not .

Link-Spirit committed:
Update Selection created , Allowing Task to be updated needs to check ID is valid or not

JCruz6725 commented:
Very comment heavy tough to follow.

JCruz6725 commented on TaskManager_ConsoleApp_Terry/App.cs:
```diff
@@ -28,8 +28,6 @@ public void Main()
             
             TaskViewer taskViewer = new TaskViewer();
```
App already declares a task viewer object. You can remove this and reuse the class member.

JCruz6725 commented on TaskManager_ConsoleApp_Terry/App.cs:
Like the task viewer above, you can make this a class member so that it is accessible for all of the app class.

JCruz6725 commented on TaskManager_ConsoleApp_Terry/TodoManager/TodoManager.cs:
```diff
@@ -44,12 +44,24 @@ public TodoItem DeleteTodo(int Id)
 
         }
-        public void UpdateStatus(int Id,Status Status)
+        public void UpdateStatus(int Id,Status status)    // can i make this method a bool ? 
```
Incomplete implementation. Please remove test code.

JCruz6725 reviewed (commented):
Needs work.

jacieylyn reviewed (commented):
A little hard to follow. 
I also test ran the code and I could not exit the program after creating? I don't know how I am supposed to update after creating a to-do item if I cant exit the create screen?
I also saw a little typo in the TaskViewer (line 38) fyi.

Link-Spirit committed:
Main ( ) ... removed duplicated code.
also Completed Implementation of Public Bool UpdateStatus
also fixed Typo in TaskViewer

JCruz6725 commented on TaskManager_ConsoleApp_Terry/App.cs:
Fix the spacing on the case statements: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/selection-statements#the-switch-statement

JCruz6725 reviewed (commented):
Looks fine. Fix the spacing as describe in the comment and its good to go.

Link-Spirit committed:
Case Statements (Spacing) Fixed

JCruz6725 reviewed (approved):
Looks fine.
